ctf/check.cmd

```diff
@@ -0,0 +1,12 @@
+@echo off
```

Comment:
Why?
With OS scripts it gets much easier to run test suites and to see their feedback.
You are going to run each suite with induvidual manually-written command?